src/screens/Assistant/ResultItem.tsx

```diff
@@ -51,37 +54,40 @@ function getFromLeg(legs: Leg[]) {
   const found = legs.find(legWithQuay);
   const fromQuay = (found?.fromEstimatedCall ?? found?.fromPlace)?.quay;
   if (!fromQuay) {
-    return legs[0].fromPlace.name ?? 'ukjent holdeplass';
+    return legs[0].fromPlace.name;
   }
   const publicCodeOutput = fromQuay.publicCode ? ' ' + fromQuay.publicCode : '';
   return fromQuay.name + publicCodeOutput;
 }
 const ResultItemHeader: React.FC<{
   tripPattern: TripPattern;
 }> = ({tripPattern}) => {
-  const quayName = getFromLeg(tripPattern.legs);
   const styles = useThemeStyles();
   const durationText = secondsToDurationShort(tripPattern.duration);
   const {t} = useTranslation();
+  const startTime = tripPattern.legs[0].expectedStartTime;
+  const endTime = tripPattern.legs[tripPattern.legs.length - 1].expectedEndTime;
```

Comment:
Potential undefined? Can't remember now exactly, but maybe there's always at least a walk leg. So this should be fine.

Reply:
Yeah think there always is, but might stilll consider the possibility, thanks for pointing that out